activate

```diff
@@ -72,15 +76,16 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 live=false
 
 # report in
-if ! out=$(wget -O - -q "$SCRIPT?image=$image&release=$release&vendor=$vendor&product=$product&serial=$serial&live=$live"); then
-    echo "wget failed"
+URL="${SCRIPT}?image=${image}&release=${release}&vendor=${vendor}&product=${product}&serial=${serial}&live=${live}"
```

Comment:
ok - have updated this to generate the submission and parse the response in JSON with a couple of python oneliners